src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js

```diff
@@ -589,19 +589,19 @@ it('Verify how many work item type exists in drop down - desktop', function() {
       expect(page.workItemDescription(page.firstWorkItem)).toBe("");
     });
 
-    it('Check before while adding description the field should not display undefine/null', function () {
-        testSupport.setBrowserMode("desktop");
-        var workItemTitle = "The test workitem title";
-        browser.wait(until.elementToBeClickable(page.detailedDialogButton), constants.WAIT, 'Failed to find Detailed Dialog Button');   
-        page.clickDetailedDialogButton();
-        var detailPage = page.clickDetailedIcon("userstory");
-
-        browser.wait(until.elementToBeClickable(detailPage.workItemDetailTitle), constants.WAIT, 'Failed to find workItem Detail Title');   
-        detailPage.setWorkItemDetailTitle (workItemTitle, false);
-
-        detailPage.clickWorkItemTitleSaveIcon();
-        expect(detailPage.workItemDetailDescriptionById().getText()).not.toBe("undefined");
-     });
+//    it('Check before while adding description the field should not display undefine/null', function () {
+//        testSupport.setBrowserMode("desktop");
+//        var workItemTitle = "The test workitem title";
+//        browser.wait(until.elementToBeClickable(page.detailedDialogButton), constants.WAIT, 'Failed to find Detailed Dialog Button');   
+//        page.clickDetailedDialogButton();
+//        var detailPage = page.clickDetailedIcon("userstory");
+//
+//        browser.wait(until.elementToBeClickable(detailPage.workItemDetailTitle), constants.WAIT, 'Failed to find workItem Detail Title');   
+//        detailPage.setWorkItemDetailTitle (workItemTitle, false);
+//
+//        detailPage.clickWorkItemTitleSaveIcon();
```

Comment:
Thanks Naina! I will make that change - if we have this situation again, you should approve the changes, and then make your own change after that - so that we save time. Thanks!